config.toml

```diff
@@ -30,7 +30,7 @@ pygmentsStyle = "tango"
  #images = ["images/project-illustration.png"]
 
  # Docsy: Google Custom Search Engine ID. Remove or comment out to disable search.
-#gcs_engine_id = "011737558837375720776:fsdu1nryfng"
+gcs_engine_id = "0007239566369470735695:624rglujm-w"
```

Comment:
Move this line (and its comment) down to the [params] section (under # Site params) or Hugo won't pick it up as a site parameter.

Reply:
Thanks @LisaFC !